sdk/go/src/sawtooth_sdk/messaging/connection.go

```diff
@@ -76,7 +76,6 @@ func NewConnection(context *zmq.Context, t zmq.Type, uri string) (*Connection, e
 
 	identity := GenerateId()
 	socket.SetIdentity(identity)
-	logger.Debug("Socket Identity set to ", identity)
 
```

Comment:
For anyone using the SDK, it generates a lot of logging output unrelated to what they are trying to use it for. The alternative would be to put more work into the sdk/logging package so that log levels can be set per-package or per-file.